src/components/GamePlayForm.js

```diff
@@ -15,11 +15,28 @@ const GamePlayForm = ({ players }) => {
     return <option key={players.indexOf(player)}>{player}</option>
   })
 
+  const submitWinner = e => {
```

Comment:
Do you have error handling if a game and/or winner is not selected before clicking submit?